features/202-session-management/config.md

```diff
@@ -33,19 +32,26 @@ The value of key `clients` is a list of client configuration:
 - `api_key`: API key.
 - `session_transport`: The transport method of session tokens.
                        Can be `cookie` or `header`.
-
-The value of key `session` is a map of session configuration:
 - `access_token_lifetime`: The lifetime of access token in seconds, default
                            to 1800.
-- `refresh_token_disabled`: Indicate whether refresh token is disabled, default
+- `refresh_token_disabled`: (valid for `header` transport only)
+                            Indicate whether refresh token is disabled, default
                             to `false`. If `session_transport` is `cookie`,
                             refresh token is disabled and this configuration
                             has no effect.
-- `refresh_token_lifetime`: The maximum lifetime of refresh token in seconds,
+- `refresh_token_lifetime`: (valid for `header` transport only)
+                            The maximum lifetime of refresh token in seconds,
                             default to max(`access_token_lifetime`, 86400).
                             Must greater than or equal to `access_token_lifetime`.
 - `session_idle_timeout_enabled`: Indicate whether session idle timeout is
                                   enabled, default to `false`.
 - `session_idle_timeout`: The session idle timeout in seconds,
                           default to min(`access_token_lifetime`, 300).
                           Must less than or equal to `access_token_lifetime`.
+- `same_site`: (valid for `cookie` transport only)
+               The `SameSite` property of cookie. Can be `lax`, `strict`, or
+               empty string. Default to `lax`.
```

Comment:
Due to golang zero value convention, we cannot use empty string to mean turning this off. Let 's introduce `disabled`